client/src/components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FunctionComponent, useState, useRef, MouseEvent } from 'react';
+
+// Styles
+import * as S from './Carousel.styles';
+
+interface CarouselState {
+  isPressed: boolean;
+  startX: number | null;
+  x: number | null;
+}
+
+interface CarouselProps {
+  setCarouselChildPointerEv: (bool: boolean) => void;
```

Comment:
would be better to rename `bool` variable so another developer could easily understand what exactly should be passed when calling the function